routes/home.js

```diff
@@ -49,5 +51,26 @@ function get(request, response) {
   response.send(html);
 }
 
+// function post(req, res) {
+//   console.log(req.body);
+//   return null;
+// }
+
+function post(request, response) {
+  const insert_user = /*sql*/ `
+      INSERT INTO users(username, age, base_flavour, fandom) VALUES($1, $2, $3, $4)
```

Comment:
This looks super tidy and easy to read. Thank you for finishing this function off.

Reply:
Thank you ✨